jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/exceptionmappers/FailureExceptionMapperSpec.scala

```diff
@@ -0,0 +1,197 @@
+package com.cerner.beadledom.jaxrs.exceptionmappers
+
+import com.cerner.beadledom.jaxrs.models.JsonError
+import com.cerner.beadledom.jaxrs.provider.{FakeDao, FakeResource}
+
+import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider
+
+import org.jboss.resteasy.mock.{MockDispatcherFactory, MockHttpRequest, MockHttpResponse}
+import org.jboss.resteasy.spi.{Failure, LoggableFailure, ReaderException, WriterException}
+import org.mockito.Mockito
+import org.mockito.Mockito._
+
+import play.api.libs.json.Json
+
+import javax.ws.rs.core.HttpHeaders._
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response.Status._
+
+/**
+  * Unit tests for {@link FailureExceptionMapper}.
```

Comment:
The syntax in scala to link to a class looks like:

```scala
/**
  * Unit tests for [[FailureExceptionMapper]].
  . . .
  */
```

Reply:
https://github.com/cerner/beadledom/pull/45/commits/865a017a55ce93a2c50c539260b17f6bbcdfe110#diff-0734403a349f9724fd379e4bead2c7e8R20